ci/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.3"
+services:
+  rw-build-env:
+    image: public.ecr.aws/x5u3w5h6/rw-build-env:latest
```

Comment:
How is this image built?

Reply:
I have made a build env image, and it can just pull the image directly when running ci, so I will comment it here.

Reply:
Is this image built from above Dockerfile?